src/components/common/TmSessionSignUpPassword.vue

```diff
@@ -111,10 +111,13 @@ export default {
     }
   },
   methods: {
+    prefix() {
+      return this.$route.params.prefix
```

Comment:
how about passing along the network id. then here we do another lookup for network (using the same gql query). from that object you now extract the prefix.

Reply:
Although computationally speaking it is better the URL solution, no?

This way we don't have weird URLs... but we still need to fetch twice or three times for the same thing along the flow :woman_shrugging:

Reply:
apollo caches the data, so there is no refetch (if the apollo queries have `fetchPolicy: "cache-first",` set)

Reply:
Thank you, I didn't know that